tensorzero-core/src/inference/types/mod.rs

```diff
@@ -120,18 +120,24 @@ impl InputMessage {
 impl InputMessageContent {
     pub async fn resolve(
         self,
+        role: Role,
```

Comment:
can you add a comment that we can drop this parameter once we deprecate role-based templates?